fulp_modules/features/antagonists/bloodsuckers/code/structures/bloodsucker_crypt.dm

```diff
@@ -479,6 +495,23 @@
 		nearly_people.adjust_hallucinations(5 SECONDS)
 		nearly_people.add_mood_event("vampcandle", /datum/mood_event/vampcandle)
 
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+// Candelabrum - Drains the sanity of non-bloodsuckers/non-vassals near it, has the brightness of a lightbulb
+/obj/structure/bloodsucker/lighting/candelabrum
+	name = "candelabrum"
+	desc = "It burns slowly and doesn't radiate heat."
+	icon_state = "candelabrum"
+	active_light_range = 3
+
+// Brazier - An alternate version of the candelbraum with roughly the brightness of a bonfire
+/obj/structure/bloodsucker/lighting/brazier
```

Comment:
I don't think this should have the same effect as the Candelabrum, I've got potential ideas for what it can do but for now I think it would be good if you separated the effects of each onto the subtypes, they share light but insanity could be candelabrum's thing and brazier can do another effect.

Reply:
The effect has been removed from the brazier. Though, out of curiosity, what new effect(s) for it do you have in mind?

Reply:
I was thinking of letting people buff their clan ability with it, like Nosferatu can temporarily gain the ability to ventcrawl, Tremere can unlock an ability's ascension or even use it as the place to upgrade powers at, etc.

Reply:
That sounds good! Please just remember that Bloodsuckers are likely to make multiple braziers for style points/aesthetic reasons. Given that, I think it shouldn't do anything overly specific, and it would be nice if the effect could stack with the number of braziers built. Above all else, please ensure that it's _fun_— thank you.